civihr_employee_portal/views/templates/views-view-table--Tasks--block.tpl.php

```diff
@@ -150,9 +154,23 @@ function _get_task_filter_by_date($date) {
                     <?php $class = 'task-row task-filter-id-' . _get_task_filter_by_date($row['activity_date_time']) . ' ' . $rowType; ?>
                     <tr id="row-task-id-<?php print strip_tags($row['id']); ?>" <?php if ($row_classes[$row_count] || $class) { print 'class="' . implode(' ', $row_classes[$row_count]) . ' ' . $class . '"';  } ?> data-row-contacts="<?php print $contactsFilterValues[strip_tags($row['task_contacts'])]; ?>">
                         <?php foreach ($row as $field => $content): ?>
-                            <?php if ($field == 'task_contacts' || $field == 'task_contacts_1' || $field == 'task_contacts_2' || $field == 'activity_date_time'):
+                            <?php if ($field == 'task_contacts' || $field == 'task_contacts_1' || $field == 'task_contacts_2'):
                                 continue;
                             endif; ?>
```

Comment:
I don't think you need to close the <?php tag here just to open it in the next line ;]